cli/napi/js_native_api.rs

```diff
@@ -1631,7 +1631,7 @@ fn napi_delete_property(
 // TODO: properly implement ref counting stuff
 #[napi_sym::napi_sym]
 fn napi_delete_reference(env: *mut Env, _nref: napi_ref) -> napi_status {
-  let mut _env = &mut *(env as *mut Env);
```

Comment:
I think these lines can all go away -- they should not have any effect.

Reply:
> I think these lines can all go away -- they should not have any effect.

Ok, I remove the unused code.